test/test/return_params_abi.cpp

```diff
@@ -10,16 +10,6 @@ namespace winrt
 
 using namespace winrt;
 
-namespace
-{
-    hstring to_hstring(::IInspectable* raw)
```

Comment:
I assume the intent of this was "does it compile?" - so probably want to keep, but remove from anonymous namespace

Reply:
That might be the case.  There was another like that which I preserved.  I misread this one as an unused helper instead of a "does it compile" test case.  I'll restore it with a suppression.